legacy-common/src/main/java/eu/arrowhead/legacy/common/model/LegacyServiceRegistryEntry.java

```diff
@@ -26,18 +28,20 @@
 	public LegacyArrowheadService getProvidedService() { return providedService; }
 	public LegacyArrowheadSystem getProvider() { return provider; }
 	public String getServiceURI() { return serviceURI; }
-	public String getServiceUri() { return serviceURI; }
```

Comment:
Do not remove this. As described in the previous pull request both getter is necesarry for proper parsing as others may use it with lower case based on the 4.1.2 client skeleton.